CMakeLists.txt

```diff
@@ -1526,6 +1525,11 @@ else()
     target_compile_definitions(open62541-plugins PRIVATE -DUA_DYNAMIC_LINKING_EXPORT)
     set_target_properties(open62541-plugins PROPERTIES FOLDER "open62541/lib")
 
+    target_compile_options(open62541-plugins PUBLIC "$<$<AND:$<COMPILE_LANGUAGE:C>,$<CONFIG:DEBUG>>:${DEBUG_TARGET_COMPILE_OPTIONS}>")
```

Comment:
There is a lot of repetition between the compile options for the different target. Please define once in a string and reuse.

is there a link to get information on the string formatting and escaping technique? -> document.